mungegithub/mungers/publish_scripts/clientgo_publish.sh

```diff
@@ -43,6 +44,23 @@ if git diff --cached --exit-code &>/dev/null; then
     exit 0
 fi
 git commit -m "${MESSAGE}"
+
+# Run "godep restore" to restore dependencies. Because entries for
```

Comment:
Is there a reason why we have to do this here and can't just fix the vendor directory in the staging directory?